src/components/override-sdk/infra/Assignment/Assignment.tsx

```diff
@@ -98,6 +98,21 @@ export default function Assignment(props) {
     setServiceShutteredStatus(serviceShuttered);
   }, [serviceShuttered]);
 
+  // Sets the language for the texts and emails if the user changes the language before opening an existing claim.
+  function initialLanguageCall() {
+    const config = { en: 'SwitchLanguageToEnglish', cy: 'SwitchLanguageToWelsh' };
+
+    thePConn
+      .getActionsApi()
+      .openProcessAction(config[lang], { caseID: thePConn.getCaseInfo()?.getKey(), type: 'Case' });
+  }
+
+  useEffect(() => {
+    if (thePConn.getCurrentClassID() === 'HMRC-ChB-Work-Claim') {
```

Comment:
to avoid this condition

Reply:
See above. But am looking into using the util 'isEduStartJourney()'